python/samples/concepts/plugins/openai_function_calling_with_custom_plugin.py

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) Microsoft. All rights reserved.
 
+from __future__ import annotations
```

Comment:
I think we have another PR where this type of import is being removed.

Reply:
Yeah, I had removed it, and I don't think I pushed my change.